www/src/templates/template-api-markdown.js

```diff
@@ -82,6 +89,33 @@ export default function APITemplate({ data, location, pageContext }) {
   const mergedFuncs = mergeFunctions(data, pageContext)
   const description = page.frontmatter.description || page.excerpt
 
+  const getAPIItems = () => {
+    return {
+      title: "APIs",
+      url: '#APIs',
```

Comment:
Can you extract the name of the id to a variable so it's clear what what the ID is being used for?

Reply:
Sure!